src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -83,6 +83,7 @@ const MESH_SILHOUETTE_RENDERING_JSON_KEY = 'meshSilhouetteRendering';
 const LINKED_SEGMENTATION_GROUP_JSON_KEY = 'linkedSegmentationGroup';
 const LINKED_SEGMENTATION_COLOR_GROUP_JSON_KEY = 'linkedSegmentationColorGroup';
 const SEGMENT_DEFAULT_COLOR_JSON_KEY = 'segmentDefaultColor';
+// const HIGHLIGHT_COLOR_JSON_KEY = 'highlightColor';
```

Comment:
I have this commented out because multicut overrides the highlightColor, so I save the previous highlightColor (as I do with a few other values and they would have similar issues) so it can be restored when multicut tool is closed. The issue is that if neuroglancer is refreshed, those prior values are lost. The easiest thing in this case was to just not have highlightColor be part of the state. The robust solution would be also adding tempHightlightColor, tempSegmentStatedColors, etc.